feature_engine/selection/__init__.py

```diff
@@ -5,7 +5,7 @@
 from .drop_correlated_features import DropCorrelatedFeatures
 from .drop_duplicate_features import DropDuplicateFeatures
 from .drop_features import DropFeatures
-from .drop_psi_features import DropHighPSIFeatures
+from .information_value import SelectByInformationValue
```

Comment:
I probable was not clear when I asked to remove this file. We need to checkout the version in master, so this file completely disappears from this PR.

Reply:
I asked in Q2 in my comment below what is the appropriate way to resolve this issue. If I checkout the commit hash that is one prior to the commit that entailed deleting DropHighPSIFeatures and its dependencies, then I would lose all of the work that I've already completed, right?

As an FYI, the commit has is `7f687d467ab0a83c91d02ce519430992638c39aa`

Sorry for the mistake!